src/main/java/uk/gov/hmcts/darts/courthouse/exception/CourthouseApiError.java

```diff
@@ -15,7 +15,7 @@ public enum CourthouseApiError implements DartsApiError {
         "Provided courthouse name already exists."
     ),
     COURTHOUSE_CODE_PROVIDED_ALREADY_EXISTS(
-        "101",
+        "100",
```

Comment:
i think you have rebased incorrectly there. should be 101

Reply:
Ye I think rebase has gone wrong :(